var/spack/repos/builtin/packages/py-enum34/package.py

```diff
@@ -29,15 +29,13 @@ class PyEnum34(PythonPackage):
     """Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4."""
 
     homepage = "https://pypi.python.org/pypi/enum34"
-    url      = "https://pypi.python.org/packages/source/e/enum34/enum34-1.1.6.tar.gz"
 
     version('1.1.6', '5f13a0841a61f7fc295c514490d120d0',
             url="https://pypi.python.org/packages/bf/3e/31d502c25302814a7c2f1d3959d2a3b3f78e509002ba91aea64993936876/enum34-1.1.6.tar.gz")
 
-    depends_on('python@2.4:2.8,3.3:')
+    depends_on('python@2.4:3.3.99')
```

Comment:
We may want to remove the version constraints altogether then. Just because it isn't needed in newer versions of Python doesn't mean that someone might not want to test a pre-release newer than what comes with python.

Yes, `:3.3` should include `3.3.1`. I'm away from my computer right now, but I'll verify this later. If not it's a bug :smiley: